war/.gitignore

```diff
@@ -1,2 +1,10 @@
-work
+work
```

Comment:
@jglick I assume it was added so as not to pollute the top level `.gitignore` with entries that are only relevant to this module.